device/transport/mqtt/devdoc/mqtt_requirements.md

```diff
@@ -67,6 +67,8 @@ The `Mqtt` and `MqttWs` constructors initialize a new instance of the MQTT trans
 
 **SRS_NODE_DEVICE_MQTT_18_025: [** If the `Mqtt` constructor receives a second parameter, it shall be used as a provider in place of mqtt.js **]**
 
+**SRS_NODE_DEVICE_MQTT_16_081: [** The `Mqtt` constructor shall subscribe to the `MqttTwinClient` `twinDesiredPropertiesUpdates` and reemit them. **]**
```

Comment:
>reemit them. [](start = 133, length = 12)

Could this be better phrased?  The verb reemit is not particularly clear.  Maybe the reemit needs to be a separate SRS?